viewer/src/views/threejs/cad/picking.ts

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Copyright 2020 Cognite AS
+ */
+
+import * as THREE from 'three';
+import { CadNode } from './CadNode';
+
+type PickDelegate = (scene: THREE.Scene, camera: THREE.PerspectiveCamera, event: MouseEvent) => number;
+
+export class CadPicker {
+  private readonly _picker: {
+    pick: PickDelegate;
+  };
+
+  constructor(renderer: THREE.WebGLRenderer, cadNode: CadNode) {
+    this._picker = createCadPicker(renderer, cadNode);
```

Comment:
Why not just implement functions inside class? #nitpick

Reply:
Just trying to avoid large classes where it is hard to see if calling a function causes mutations or not. That said, closures do not really help in this regard. I will move the implementation into a free function instead to make it more evident.